plugins/flatpak/gs-flatpak.c

```diff
@@ -3972,6 +4025,14 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
+	/* Check disk space requirements for the list */
```

Comment:
Instead of having this comment, I think it'd be better to rename the `...allow_install_or_update` to `gs_flatpak_has_space_to_install_or_update`. Self-documented, so no need for that comment.